.changeset/tender-toys-think.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Many thanks for the contribution @lll-lll-lll-lll and good job spotting that! 

The empty changeset is fine in this case, could you please remove this file (`tender-toys-think`)? Then this PR is good to go :)

Reply:
@nikosdouvlis ok! I removed it!